app/services/middleware_topology_service.rb

```diff
@@ -1,30 +1,27 @@
 class MiddlewareTopologyService < TopologyService
+  include UiServiceMixin
```

Comment:
@Jiri-Kremser I was going to, but for now there are some issues with the way I intended to add things to the `UiServiceMixin`, since need to distinguish by type, for MW servers (Hawkular, EAP or Wildfly) and for MW Deployments (WAR, EAR, etc) as we have in the `middleware_*_decorator.rb` files.
For now will just fix the tests for this and work on this in a separate PR.